akka-projection-kafka/src/it/scala/akka/projection/kafka/integration/KafkaToSlickIntegrationSpec.scala

```diff
@@ -228,20 +230,63 @@ class KafkaToSlickIntegrationSpec extends KafkaSpecBase(ConfigFactory.load().wit
               val eventType = envelope.value()
               val userEvent = UserEvent(userId, eventType)
               // do something with the record, payload in record.value
+              println(s"# process ${envelope.offset()}: $userId $eventType") // FIXME
               repository.incrementCount(projectionId, userEvent.eventType)
             })
       }
 
-      projectionTestKit.run(slickProjection(), remainingOrDefault) {
+//      projectionTestKit.run(slickProjection(), remainingOrDefault) {
+//        assertEventTypeCount(projectionId)
+//        assertAllOffsetsObserved(projectionId, topicName)
+//      }
+
+      val projection1 = testKit.spawn(ProjectionBehavior(slickProjection()))
```

Comment:
using the `ProjectionBehavior` here is just temporary to show that it can be stopped and resumed, since that was discussed in https://github.com/akka/akka-projection/pull/424